syntaxes/inmanta.json

```diff
@@ -129,6 +129,21 @@
     {
       "include": "#special"
     },
+    {
+      "patterns": [
+        {
+          "match": "(matching)[\\s]+(/(?:[^/\\\\\\n]|\\\\.)+/)",
```

Comment:
Yes. That's normal. We have two rules for the matching keyword:

* We have one rule that matches `matching <regex>`. This rule is required because a regex is demarcated by two slashes. Without this rule the lexer could incorrectly consider the slash a division operator. 
* We have another rule that matches just the "matching" keyword. This one handles all the other situation where the matching keyword is used without a regex. For example `typedef test as string matching self < 1000`